### PR TITLE
Fix filepath in tex file on Windows.

### PR DESCRIPTION
**Pull request by fredrikekre**
See https://discourse.julialang.org/t/issue-with-documenters-pdf-output-pictures-not-found/23023/

### CODE REVIEW

File: src/Writers/LaTeXWriter.jl [hunk 1]
```diff
@@ -533,7 +533,7 @@ function latexinline(io::IO, md::Markdown.Image)
             # URLs starting with a / are assumed to be relative to the document's root
             normpath(lstrip(md.url, '/'))
         else
-            normpath(joinpath(dirname(io.filename), md.url))
+            normpath(dirname(io.filename) * "/" * md.url)
```
Comment (fredrikekre) [inline src/Writers/LaTeXWriter.jl]: I guess there could also be `\\` in `dirname(io.filename)` and `md.url`? Maybe safest to do this:
```suggestion
            normpath(replace(dirname(io.filename) * "/" * md.url, "\\" => "/"))
```

**Comment by cserteGT3** [comment]
With the latest version (` "\\" => "/" `) it's working.
(I'm still getting some error, but now I have a pdf with the pictures).
Thank you for the quick fix!

**Comment by mortenpi** [comment]
LGTM. No need for a backport, right? Can just update changelog and `Project.toml` here, merge and tag?

**Comment by fredrikekre** [comment]
> No need for a backport, right? Can just update changelog and Project.toml here, merge and tag?

Ah, right. Done.

**Pull request merged**